etl/steps/data/garden/energy/2022-09-09/shared.py

```diff
@@ -0,0 +1,98 @@
+from pathlib import Path
+from typing import List
+
+import pandas as pd
+from owid import catalog
+
+CURRENT_DIR = Path(__file__).parent
+VERSION = CURRENT_DIR.name
+
+
+def gather_sources_from_tables(
```

Comment:
This look pretty useful - can we move it to `grapher_helpers.py` so that others can reuse it?

Reply:
OK, we can do that in another PR.